plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlPlugin.java

```diff
@@ -24,4 +27,10 @@ public PostgreSqlPlugin()
     {
         super("postgresql", combine(new PostgreSqlClientModule(), new PostgreSqlConnectionFactoryModule()));
     }
+
+    @Override
+    public Set<Class<?>> getFunctions()
+    {
+        return ImmutableSet.of(VectorFunctions.class);
```

Comment:
Any particular reason why these functions need to be in postgresql connector instead of the engine ?

Reply:
Ya, these functions should be in the main engine